### DISCUSSION TITLE
Suggestion on deploying operator in multiple namespaces

### DISCUSSION THREAD

**Discussion by prakashpsgcse** [strimzi/strimzi-kafka-operator#10180] [General]
We have k8s cluster with 5 namespaces and we deployed single operator to watch all namespace. We would like to get a suggestion on best practice on operator deployment when we go for PodSetController.In past we faced issue where all 5 clusters were down  with /tmp size issues . We increased /tmp dir size but operator took sometime to apply /tmp changes in StateFullSet of Kafka . During that phase reconciliation was slow (it was like reconciliation of namespace one by one). 

We would like to understand some functionality of cluster operator and need suggestion on operator deployment with enabled UseStrimziPodSets.

1. Does cluster operator reconciles all watch namespaces parallel or it reconciles one and picks next one ?  
2. What was the suggested deployment option for cluster operator when we have 5 namespace. Each namespace we have 5-zk,10-kafka,5 -Connect , 5- mirror . totatl each namespace we have 25 pods . With 5 namespace we have around 150 pods. Deploying one cluster operator to watch all 5 namespace is recommended or we should deploy operator per namespace ?

**Comment by scholzj** [comment]
The operator always reconciles the different clusters in parallel if needed. And always did. So not sure what exactly was the situation you observed. In general, having a cluster-wide operator installation is probably the best thing for you.
